apps/gallery/js/info.js

```diff
@@ -52,10 +52,30 @@ function showFileInformation(fileinfo) {
 
       // Populate info overlay view
       MediaUtils.populateMediaInfo(data);
+      // Hide Resolution for video files. See Bug 1217989
+      setFieldVisibility('info-resolution', fileinfo);
     });
   }
 
   function getFileName(path) {
     return path.split('/').pop();
   }
+
+  function setFieldVisibility(id, fileinfo) {
```

Comment:
nit: this would be a cleaner api if you pass a boolean as the second argument, and check fileinfo.metadata.video above where you call it instead of inside this function.